luskan/src/main/java/org/opencadc/luskan/ws/StorageTapService.java

```diff
@@ -252,4 +254,17 @@ private String getState() {
         return ret;
     }
 
+    // Check that the TempStorageManager results directory is writable.
```

Comment:
If you were to put this method into LuskanConfig and call it from initConfig(), then the check would also run on startup and throw an exception -- not just in the availability.

Reply:
That's a better place for that check, moved to LuskanConfig.